docs/cfn/AWS_TESTING.md

```diff
@@ -0,0 +1,74 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# AWS integration testing
+
```

Comment:
why `/docs/cfn/` ? What is `cfn`?

Reply:
@zeroshade good question, it was currently a bit adhoc, and mostly my "notes" and code related to manaul testing so I put it in docs.

Happy to move it up a level, it was my view that it would probably form some examples or reference infrastructure at some point.

Again interested in your thoughts, I like to provide lots of resources to consumers, giving them as much as possible to ensure they succeed.